src/react/components/pages/editorPage/canvas.tsx

```diff
@@ -283,12 +289,27 @@ export default class Canvas extends React.Component<ICanvasProps, ICanvasState>
         if (this.props.onSelectedRegionsChanged) {
             this.props.onSelectedRegionsChanged([]);
         }
+        if (selectedRegions.find((r) => r.category !== selectedRegions[0].category)
```

Comment:
according to our discussion. I think it should be:
if (selectedRegions.length === 1 && selectedRegions[0].category === RegionCategory.Checkbox) {
    this.setTagType(inputTag[0], FieldType.Checkbox);
}

Reply:
Since right now we don't add just one checkbox element can be added in tag, then if we make this change, 2 checkboxes under one tag, type would also be string. Is that ok?